app/models/lesson.rb

```diff
@@ -0,0 +1,5 @@
+class Lesson < ApplicationRecord
+  has_and_belongs_to_many :terms
+  validates_presence_of :body, :lesson_type, :questions, :title
+  validates_inclusion_of :lesson_type, in: ["Activity-based", "Article-based"]
```

Comment:
I would use snake case for the types : `[:activity_based, :article_base]` (or maybe `['activity-based', 'article_based'] ?)